lib/Gitlab/Api/Jobs.php

```diff
@@ -0,0 +1,133 @@
+<?php namespace Gitlab\Api;
+
+use Psr\Http\Message\StreamInterface;
+
+class Jobs extends AbstractApi
+{
+    const SCOPE_CREATED = 'created';
+    const SCOPE_PENDING = 'pending';
+    const SCOPE_RUNNING = 'running';
+    const SCOPE_FAILED = 'failed';
+    const SCOPE_SUCCESS = 'success';
+    const SCOPE_CANCELED = 'canceled';
+    const SCOPE_SKIPPED = 'skipped';
+    const SCOPE_MANUAL = 'manual';
+
+    /**
+     * @param int|string $project_id
+     * @param array $scope
+     * @return mixed
+     */
+    public function jobs($project_id, array $scope = [])
```

Comment:
The name `all` seems to be more used than the plural noun form. Could you change the method name to `all`?